AudioChannel.js

```diff
@@ -108,6 +108,11 @@ AudioChannel.prototype.playLoopSound = function (soundId, volume, pan, pitch) {
 		sound.play(volume * self.volume, pan, pitch); // load and play
 	}
 
+	function playNextSound() {
+		// force loading to happend in a new thread in order to let Garbage Collector to release previous audio.
```

Comment:
Also, with regards to GC, it may really help to set `this.loopSound` to null right after calling stopCurrentLoop. That would make `sound` inside `_playNextSound` the last remaining reference (if I'm not missing anything), which would hopefully cause optimistic garbage collection right after the callback of `sound.stop` has been run. You could even set `sound` to null there before calling the callback. Then there are no references left and GC *may* occur instantly.